ddtrace/contrib/pyramid/trace.py

```diff
@@ -10,6 +10,7 @@
 from ...internal.logger import get_logger
```

Comment:
We can remove pyramid from the black ignores in pyproject.toml (you might also have to format the rest of the files in ddtrace/contrib/pyramid/ by doing `black ddtrace/contrib/pyramid/*.py`